cirq/protocols/json_serialization.py

```diff
@@ -140,6 +140,7 @@ def two_qubit_matrix_gate(matrix):
         'PhaseFlipChannel': cirq.PhaseFlipChannel,
         'PhaseGradientGate': cirq.PhaseGradientGate,
         'PhasedFSimGate': cirq.PhasedFSimGate,
+        'PhasedFSimParameters': cirq.google.PhasedFSimParameters,
```

Comment:
If you want, you can add this to the NOT_YET_SERIALIZED list, and I can add the json serialization if you want?

Reply:
There is no problem with json serialization for this dataclass as far as I know (I'll double-check this). The problems are in PR 2)